src/sentry/static/sentry/app/components/sidebar/index.jsx

```diff
@@ -411,6 +426,11 @@ class Sidebar extends React.Component {
         {hasOrganization && (
           <SidebarSectionGroup>
             <SidebarSection>
+              {HookStore.get('sidebar:bottom-items').length > 0 &&
```

Comment:
Gonna try and look into why we can't use the Hook component here 😢

Reply:
That would be great - it's definitely cleaner to use `Hook`. It hasn't come up for anything else i've used it for so I was surprised by the behaviour. Once we get that fixed up and refactor to add `default` to it we can swap it in